.travis.yml

```diff
@@ -49,6 +49,19 @@ jobs:
       env: PURPOSE='Automation'
       script: ./scripts/ci/test_automation.sh
       python: 2.7
+    - stage: verify
+      env:
+      - PURPOSE='Automation'
+      - REDUCE_SDK='True'
+      - AZURE_CLI_TEST_TARGET_PROFILE='2017-03-09'
+      script: ./scripts/ci/test_automation.sh
+      python: 3.6
+    - stage: verify
+      env:
+      - PURPOSE='Automation'
+      - AZURE_CLI_TEST_TARGET_PROFILE='2018-03-01'
+      script: ./scripts/ci/test_automation.sh
+      python: 2.7
```

Comment:
Why is one run on 2.7 and the other on 3.6? I'm fine with the split, but I'm wondering if there is any inner consideration?

Reply:
I just replicated how latest tests are setup. I think it gives us coverage on both the versions of python.